### ISSUE TITLE
Parcels aka Projects

### ISSUE THREAD

**Issue by emilyhobson** [stamen/bluegreenway#3]
List of parcels that need to be called out on the Parcels/Projects map.

https://docs.google.com/a/spur.org/spreadsheets/d/1HIRLc2usDhXLP_a9nbboAaalNiHga4lGt025616TI0k/edit?usp=sharing

**Comment by almccon** [comment]
Current state of that file: 

| Property | Parks/parcels |
| --- | --- |
| Seawall lot 337/Mission Rock |  |
|  | China Basin Park |
| Arena |  |
|  | Bayfront Park |
| Aguavista Park and Fishing Pier |  |
| Pier 70 |  |
|  | waterfront park |
| Warm Water Cove |  |
| Islais Creek |  |
| Pier 94 Wetlands |  |
| India Basin |  |
|  | Heron's Head Park |
|  | PG&E |
|  | India Basin Shoreline Park |
|  | India Basin Open Space |
| Hunters Point Shipyard |  |
| Yosemite Slough |  |
| Candlestick Point Recreation Area |  |

I need shapefiles or some official geodata for these parcels. I could try to get something close enough using OpenStreetMap and/or the city's parcel data file.

**Comment by almccon** [comment]
We've got two shapefiles, one of parcels (from the City's point of view) and one of open spaces. They look a bit like this:
<img width="434" alt="screen shot 2016-03-22 at 1 39 19 pm" src="https://cloud.githubusercontent.com/assets/1212178/13968533/7b6f4694-f03a-11e5-96ec-dcd67ef191c8.png">

Most of the "projects" we want to map (like "Arena" for example) are presumably made up of multiple parcels. We'd need to now the parcel ID (called `mapblklot` or `blklot` in the shapefile) for all the parcels that make up each larger project.

**Comment by almccon** [comment]
For the projects that are linear features along streets, here are the appropriate extents:

Cargo (Illinois to Jennings)

Jennings (Cargo to Evans)

Evans / Hunters Point Blvd (same right-of-way / different names) from Jennings to Innes

Innes (Hunters Point Blvd to Donahue)

Hudson (running parallel to Innes one block northeast) Hunters Point Blvd to Earl

**Comment by almccon** [comment]
That description (from Mat Snyder at SF Planning) looks like this:

<img width="416" alt="screen shot 2016-05-03 at 3 may 6 44 32" src="https://cloud.githubusercontent.com/assets/1212178/15003282/32f3a478-115f-11e6-93e2-9a9d5df02548.png">

**Comment by almccon** [comment]
For the other linear features, I guess I can just eyeball them myself.

**Comment by almccon** [comment]
Here are the linear features that _I've_ identified are:
1. Terry Francois Blvd Improvements
2. Illinois Street
3. 24th Street Improvements
4. Cargo Way
5. Hudson Avenue Right-Of-Way Improvements

The five segments from Mat Synder above seem to correspond with the final two on my list. The first three, I'll just wing it.

**Comment by almccon** [comment]
Okay, I just guessed at the others:
![screen shot 2016-05-17 at 17 may 5 26 15](https://cloud.githubusercontent.com/assets/1212178/15343747/8bec8e50-1c54-11e6-811e-756e65db29fa.png)

And I gave them temporary IDs:

| name | id |
| --- | --- |
| Cargo Way | 501 |
| Jennings St | 502 |
| Evans | 503 |
| Innes | 504 |
| Hudson | 505 |
| Illinois Street | 506 |
| 24th Street | 507 |
| Terry Francois Blvd | 508 |

And here I've dissolved the southernmost four into something I'll use for "Hudson Avenue Right-Of-Way Improvements".

```
SELECT temp_id, name, the_geom_webmercator FROM "stamen-org".bgw_linear_features WHERE temp_id IN (501,506,507,508)
UNION ALL
select 510 as temp_id, 'Hudson Avenue Right-Of-Way Improvements' as name, ST_Collect(the_geom_webmercator) as the_geom_webmercator FROM "stamen-org".bgw_linear_features WHERE temp_id IN (502,503,504,505)
```

![screen shot 2016-05-17 at 17 may 5 43 50](https://cloud.githubusercontent.com/assets/1212178/15343981/f0681672-1c56-11e6-9966-544191f69ce5.png)

**Comment by almccon** [comment]
And then with a 15 meter buffer: 

```
SELECT temp_id, name, ST_Buffer(the_geom_webmercator, 15) as the_geom_webmercator
FROM (SELECT temp_id, name, the_geom_webmercator FROM "stamen-org".bgw_linear_features WHERE temp_id IN (501,506,507,508)
UNION ALL
select 510 as temp_id, 'Hudson Avenue Right-Of-Way Improvements' as name, ST_Collect(the_geom_webmercator) as the_geom_webmercator FROM "stamen-org".bgw_linear_features WHERE temp_id IN (502,503,504,505)
) a
```

![screen shot 2016-05-17 at 17 may 5 48 22](https://cloud.githubusercontent.com/assets/1212178/15344037/92f93a56-1c57-11e6-875a-6970e34c741e.png)

Saved as `bgw_linear_features_buffered`. Add to `bgw_projects` like so, and update `bgw_projects`:

```
SELECT the_geom_webmercator, bgw_id, name FROM bgw_projects
UNION ALL
SELECT the_geom_webmercator, temp_id as bgw_id, name FROM bgw_linear_features_buffered
```

Then I'll update the GeoJSON.

**Comment by almccon** [comment]
Remaining tasks:

Remove overlap for: 
- India Basin Shoreline Park

Draw shapes for:
- Hunters Point Shipyard (North and South)
- Pier 70 stuff
- 700 / 900 Innes
- PG&E Shoreline

**Comment by emilyhobson** [comment]
Can you re-point given outstanding tasks, @almccon ?

**Comment by almccon** [comment]
I stand by 5 hours

**Comment by almccon** [comment]
In #61 I'm categorizing all the locations as either "open space" or "project". But now that I look at the spreadsheet, there's perhaps a third category (which I originally thought was a section header): "property". There are only three things in that category "Seawall Lot 337", "Pier 70", and "NRG". The Pier 70 one appears to encompass many others... and the NRG appears to be another name for PG&E.

Current state of the spreadsheet

| Zone | Property | Project | Open Space |
| --- | --- | --- | --- |
| Mission Bay/ Mission Rock | Seawall lot 337/Mission Rock |  |  |
| Mission Bay/ Mission Rock |  | Terry Francois Blvd Improvements |  |
| Mission Bay/ Mission Rock |  | Chase Center Arena (Warriors) |  |
| Mission Bay/ Mission Rock |  | Illinois Street |  |
| Mission Bay/ Mission Rock |  |  | China Basin Park |
| Mission Bay/ Mission Rock |  |  | Bayfront Park |
| Mission Bay/ Mission Rock |  |  | Aguavista Park and Fishing Pier |
| Mission Bay/ Mission Rock |  |  | Pier 52 Boat Launch |
| Mission Bay/ Mission Rock |  |  | China Basin Park |
| Mission Bay/ Mission Rock |  |  | Mission Bay Parks 23 & 24 |
| Mission Bay/ Mission Rock |  |  | Mission Creek Shoreline North |
| Mission Bay/ Mission Rock |  |  | Mission Creek Shoreline South |
| Pier 70/Central Waterfront | Pier 70 |  |  |
| Pier 70/Central Waterfront |  | Waterfront Park |  |
| Pier 70/Central Waterfront |  | Pier 94 Wetlands |  |
| Pier 70/Central Waterfront |  |  | 24th Street Improvements |
| Pier 70/Central Waterfront |  |  | Islais Creek North-West |
| Pier 70/Central Waterfront |  |  | PG&E Shoreline |
| Pier 70/Central Waterfront |  |  | Power Plant Shoreline Access |
| Pier 70/Central Waterfront |  |  | Pier 64 Shoreline Access |
| Pier 70/Central Waterfront |  |  | Pier 70 Crane Cove Park |
| Pier 70/Central Waterfront |  |  | Pier 70 Slipways Park |
| Pier 70/Central Waterfront |  |  | Pier 70 Upland Open Spaces |
| Pier 70/Central Waterfront |  |  | Tulare Park/Islais Creek North-East |
| Pier 70/Central Waterfront |  |  | Warm Water Cove Park |
| India Basin | NRG | PG&E |  |
| India Basin |  | 700 Innes | aka India Basin Shoreline Park |
| India Basin |  | 900 Innes | aka India Basin Shoreline Park |
| India Basin |  | Hudson Avenue Right-Of-Way Improvements |  |
| India Basin |  |  | India Basin Shoreline Park |
| India Basin |  |  | India Basin Open Space |
| India Basin |  |  | Heron's Head Park |
| India Basin |  |  | Cargo Way |
| India Basin |  |  | Islais Landing/Islais Creek South |
| India Basin |  |  | Jennings St/Hunters Point Blvd/Innes Ave |
| India Basin |  |  | Third and Cargo Gateway |
| Shipyard/Candlestick |  |  | Candlestick Point State Recreation Area |
| Shipyard/Candlestick |  |  | Hunters Point Shipyard Open Spaces North |
| Shipyard/Candlestick |  |  | Hunters Point Shipyard Open Spaces South |
| Shipyard/Candlestick |  |  | Yosemite Slough Wetland |

**Comment by almccon** [comment]
Update here: http://studio.stamen.com/bluegreenway/blog/2016/06/07/new-complete-project-shapes/

tl;dr: I skipped Seawall Lot 337 and 700/900 Innes.

I drew all the missing shapes, and re-drew some of the shapes we already had, based on what I could see in the various PDFs.

Some of the zones assigned to the locations don't quite make sense:
- It looks like it makes more sense to include “Mission Bay Parks 23 & 24″ and “Agua Vista Park” in the Mission Bay zone instead of Pier 70/Central Waterfront. Maybe the zone polygon for Mission Bay should extend south to Mariposa Street instead of 16th?
- “Pier 94 Wetlands” in is the Pier 70/Central Waterfront zone in the spreadsheet, but it looks like it belongs in the India Basin zone if we look at the map.
- The dividing line between the India Basin Zone and Hunters Point Shipyard zone follows Innes Avenue all the way to the water, but this would put “Hunters Point Shipyard Open Spaces North” in India Basin instead of Hunters Point. So for now I left it tagged with the Hunters Point zone.

I'll track that in another issue: #78

**Comment by almccon** [comment]
900 Innes:
http://sfrecpark.org/wp-content/uploads/Final-ABCA-900-Innes-Ave-Site-Report.pdf
<img width="645" alt="screen shot 2016-06-13 at 13 jun 5 37 56" src="https://cloud.githubusercontent.com/assets/1212178/16027794/ca236e84-318d-11e6-88ed-b30ae8fff2b6.png">

**Comment by almccon** [comment]
Better map of 900 Innes: http://sfrecpark.org/wp-content/uploads/Item-10-900-Innes-Acquisition-Staff-Report-061914.pdf
<img width="652" alt="screen shot 2016-06-13 at 13 jun 5 43 02" src="https://cloud.githubusercontent.com/assets/1212178/16027835/5245ff2a-318e-11e6-842a-bd61d11f6bc8.png">

**Comment by almccon** [comment]
700 Innes: http://www.socketsite.com/archives/2013/04/15_acres_of_bay_front_property_in_distress_and_up_for_g.html
<img width="766" alt="screen shot 2016-06-13 at 13 jun 5 54 56" src="https://cloud.githubusercontent.com/assets/1212178/16028026/fc3825fc-318f-11e6-9b5b-41feaa95f7c8.png">
